index.js

```diff
@@ -128,11 +134,53 @@ async function task(options = { quiet: true }) {
   if (options.quiet) { // for running with setInterval
     process.stdout.write('.')
   } else { // for individual runs, or first of a setInterval run
-    console.log(`No available appointments for months ${MONTHS.join(', ')} in ${EMBASSY.displayName} (${EMBASSY.postCode})`)
+    console.log(`No available appointments for months ${DATES.flatMap(e => e.MONTH).join(', ')} in ${EMBASSY.displayName} (${EMBASSY.postCode})`)
+  }
+}
+
+function initialize() {
```

Comment:
If the embassy is dynamic, then initialize() needs to return some values that are then passed to the task function, rather than using mutating global objects. We can already see that DEFAULT_HEADERS.Referrer is going to be always using the CR embassy, since it is determined at the time the js script is first loaded.